sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -559,31 +579,33 @@ def size_in_bytes(self):
         return self._size
 
     def add_message(self, message):
-        # type: (Union[ServiceBusMessage, Mapping[str, Any]]) -> None
+        # type: (Union[ServiceBusMessage, AMQPAnnotatedMessage, Mapping[str, Any]]) -> None
         """Try to add a single Message to the batch.
 
         The total size of an added message is the sum of its body, properties, etc.
         If this added size results in the batch exceeding the maximum batch size, a `MessageSizeExceededError` will
         be raised.
 
         :param message: The Message to be added to the batch.
-        :type message: ~azure.servicebus.ServiceBusMessage
+        :type message: Union[~azure.servicebus.ServiceBusMessage, ~azure.servicebus.AMQPAnnotatedMessage]
         :rtype: None
         :raises: :class: ~azure.servicebus.exceptions.MessageSizeExceededError, when exceeding the size limit.
         """
 
         return self._add(message)
 
     def _add(self, add_message, parent_span=None):
-        # type: (Union[ServiceBusMessage, Mapping[str, Any]], AbstractSpan) -> None
+        # type: (Union[ServiceBusMessage, Mapping[str, Any], AMQPAnnotatedMessage], AbstractSpan) -> None
         """Actual add implementation.  The shim exists to hide the internal parameters such as parent_span."""
         message = create_messages_from_dicts_if_needed(add_message, ServiceBusMessage)
         message = transform_messages_to_sendable_if_needed(message)
         message = cast(ServiceBusMessage, message)
         trace_message(
             message, parent_span
         )  # parent_span is e.g. if built as part of a send operation.
-        message_size = message.message.get_message_encoded_size()
+        message_size = (
+            message.message.get_message_encoded_size()
+        )  # pylint: disable=protected-access
```

Comment:
Why was 'disable=protected-access' needed here?

Reply:
it's no needed, this was previously me trying to add some new apis.
fixed